src/components/timers/Countdown.tsx

```diff
@@ -0,0 +1,189 @@
+import Button from '../generic/Button';
+import DisplayTime from '../generic/DisplayTime';
+import Inputs from '../generic/Inputs';
+import Panel from '../generic/Panel';
+
+import { useEffect, useRef, useState } from 'react';
+import { getDisplayHundredths, getDisplayMinutes, getDisplaySeconds } from '../../utils/helpers';
+
+interface CountdownProps {
+    onChange?: (config: {
+        workTime: { minutes: number; seconds: number };
+        isValid: boolean;
+    }) => void;
+    newTimer?: boolean; // Determines if this is a new timer being configured
+    workoutTimer?: boolean; // Determines if this is a timer being controlled by the workout
+    workTime?: { minutes: number; seconds: number }; // Work time configuration
+    elapsedTime?: number; // Elapsed time in milliseconds provided in workout context
+    active?: boolean; // Determines if the currently active timer in a workout
+    state?: 'not running' | 'running' | 'paused' | 'completed';
+}
+
+const Countdown: React.FC<CountdownProps> = ({ workTime = { minutes: 0, seconds: 0 }, elapsedTime = 0, onChange, newTimer = false, workoutTimer = false, active = false, state = 'not running' }) => {
+    const [inputMinutes, setInputMinutes] = useState(workTime.minutes);
```

Comment:
To reduce the number of state hooks you have (you can use this in your other components, too), you could consider creating a single state hook called `inputTime` which handles seconds and minutes
`const [inputTime, setInputTime] = useState({seconds: workTime.seconds, minutes: {workTime.minutes})`